db/migrate/20181213113655_add_index_to_email_status.rb

```diff
@@ -0,0 +1,5 @@
+class AddIndexToEmailStatus < ActiveRecord::Migration[5.2]
+  def change
+    add_index :emails, :status
```

Comment:
If we do want to add this index, we should [add it concurrently](https://robots.thoughtbot.com/how-to-create-postgres-indexes-concurrently-in) to avoid locking the table.